safenode/src/network/mod.rs

```diff
@@ -234,6 +239,8 @@ fn restart_at_random(peer_id: &PeerId) {
 /// API to interact with the underlying Swarm
 pub struct Network {
     pub(super) swarm_cmd_sender: mpsc::Sender<SwarmCmd>,
+    #[allow(dead_code)]
```

Comment:
Why add it if dead code? I guess it is no longer dead, and this decorator missed to be removed?

Reply:
It is used inside the test, i.e, inside `cfg(test)` for now, so it still reports the field as dead_code. But I'm pretty sure storing the peer_id will be useful later down the line.